client/src/Component/MainTable.js

```diff
@@ -86,6 +88,8 @@ const MainTable = () => {
 								<td>{new Date(client.date_of_birth).toLocaleDateString()}</td>
 								<td>{client.postcode}</td>
 								<td>{client.referring_agency}</td>
+								<td>{new Date(client.pick_up_date).toLocaleDateString()}</td>
+								<td>{client.is_decline}</td>
```

Comment:
```suggestion
								<td>{client.is_declined}</td>
```
This should match the database column name in [create_database.sql](https://github.com/CodeYourFuture/scot-callback-cats/blob/main/server/scripts/create_database.sql#L29)

Reply:
Already changed thanks again.